mysql_con.js

```diff
@@ -41,6 +44,30 @@ router.get("/format.css", function(req, res) {
 router.get("/main.js", function(req, res) {
   res.sendFile(__dirname+"/app/webpage/main.js");
 });
+
+/* lgin-user-02 */
+// login requests --------------------------------------------------------------
+router.route("/login/user")
+
+  .post(function(req, res) {  // determening if user exists
+    posts.checkUser(res, req, con);
```

Comment:
Something I tried to say in the comment you didn't understand. I was doing a search for number of occurrences in the database, and if I use a get, then I can't send data to the query so I can specify what email to search for.

I could do a get just for the name alternately. But I was liking the privacy of just returning a number.